WhaleSpotting/Repositories/LikeRepo.cs

```diff
@@ -27,4 +29,11 @@ public void Create(LikeRequest newLikeRequest, int userId)
         });
         context.SaveChanges();
     }
+    
+    public void Delete(int likeId)
+    {
+        var like = new Like { Id = likeId };
```

Comment:
Can you get the like from the database using ID? There's a chance that it doesn't exist which means it could fail on the next line so we should handle this case

Reply:
Additional checks added